ironfish/src/blockchain/blockchain.test.ts

```diff
@@ -541,33 +541,38 @@ describe('Blockchain', () => {
     const blockA4 = await useMinerBlockFixture(nodeA.chain)
     await expect(nodeA.chain).toAddBlock(blockA4)
 
+    const blockA5 = await useMinerBlockFixture(nodeA.chain)
+    await expect(nodeA.chain).toAddBlock(blockA5)
+
     const blockB1 = await useMinerBlockFixture(nodeB.chain)
     await expect(nodeB.chain).toAddBlock(blockB1)
 
     const blockB2 = await useMinerBlockFixture(nodeB.chain)
     await expect(nodeB.chain).toAddBlock(blockB2)
 
-    const { block: blockB3 } = await useBlockWithTx(nodeB)
-    await expect(nodeB.chain).toAddBlock(blockB3)
+    // blockB3 is created and added by the fixture automatically
```

Comment:
```suggestion
    // blockB4 is created and added by the fixture automatically
```

Reply:
This suggestion makes the comment incorrect.